src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+}
+
+export const Tabs: FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  const currentTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const isActive = tab.id === currentTab.id;
+
+            return (
+              <li
+                className={cn({
+                  'is-active': isActive,
+                })}
+                data-cy="Tab"
+                key={tab.id}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (!isActive) {
+                      onTabSelected(tab);
+                    }
+                  }}
+                >
```

Comment:
move it to a new component

Reply:
I guess you are looking at the old code in this place